inventory/environ.py

```diff
@@ -117,12 +106,6 @@ def getDefaultVars():
     #If the value is set, we would use that, otherwise, return True
     defaultVars["splunk"]["preferred_captaincy"] = False if os.environ.get('SPLUNK_PREFERRED_CAPTAINCY', "").lower() == "false" else True
     defaultVars["splunk"]["hostname"] = os.environ.get('SPLUNK_HOSTNAME', socket.getfqdn())
-
-    # Lower indexer search/replication factor when indexer hosts less than 3
-    if "splunk_indexer" in inventory and "hosts" in inventory["splunk_indexer"] and len(inventory["splunk_indexer"]["hosts"]) < 3:
-        defaultVars["splunk"]["idxc"]["search_factor"] = 1
-        defaultVars["splunk"]["idxc"]["replication_factor"] = 1
```

Comment:
I removed this section because this is slightly redundant with recent changes made in https://github.com/splunk/splunk-ansible/pull/330/files